practice_app/pages/api/getStatistics.js

```diff
@@ -2,44 +2,54 @@ import axios from 'axios'
 
 const { checkApiResponseForCountry } = require('./getStatisticsModule')
 
-export default (req, res) => {
-    // req object contains whatever information we can get from the request that comes from the client (from the browser)
-    // res object is used tow write our reponse
-    const { query, method } = req
+export default (req, res) =>
+    new Promise(async resolve => {
+        // req object contains whatever information we can get from the request that comes from the client (from the browser)
+        // res object is used tow write our reponse
```

Comment:
I would suggest that you write `export default async (req, res) => ...` and remove the whole `new Promise(async resolve => { ...` part is it is clearer and does the "Promise wrapping" internally, also handles errors automatically (while with promise you would have to use both `resolve` and `reject` parameters)